internal/http/proxy/proxy.go

```diff
@@ -84,3 +90,28 @@ func modifyResponse(blockFieldSuggestions *block_field_suggestions.BlockFieldSug
 		return nil
 	}
 }
+
+func logErrors(payload map[string]interface{}, log *slog.Logger) map[string]interface{} {
+	errorsPayload := payload["errors"]
+
+	if errorsPayload != nil {
+		errorsJSON, _ := json.Marshal(errorsPayload)
+
+		var graphqlErrors []GraphqlError
+		err := json.Unmarshal(errorsJSON, &graphqlErrors)
+
+		if err == nil {
+			for _, graphqlError := range graphqlErrors {
+				log.Error("Error occurred at", "path", strings.Join(graphqlError.Path, "->"), "message", graphqlError.Message)
```

Comment:
I would log as info or warn, but not error. While we log errors this is not actually an error level log

Reply:
oh good one indeed, woeps